package.json

```diff
@@ -15,6 +15,15 @@
   },
   "homepage": "https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week3-London8#readme",
   "devDependencies": {
-    "jest": "^27.4.7"
+    "@testing-library/dom": "^8.19.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/user-event": "^13.5.0",
+    "jest": "^27.4.7",
+    "jsdom": "^20.0.1"
```

Comment:
This is pretty much a pure guess but the `TypeError: Cannot read property of undefined ( read 'navigator' )` error sounds like it might be related to the version of either Jest or JSDOM to me.